examples/scenarios/wandb_training_smoke.py

```
@@ -0,0 +1,668 @@
+                class_names=class_names,
+                row_limit=args.sample_predictions,
+            )
+
+            if args.wandb_log_attribution:
+                try:
+                    attribution_bundle_dir = _capture_attribution_bundle(
+                        output_dir,
+                        device=device,
+                        history_recorded=history_enabled,
+                    )
+                except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)